neuralprophet/benchmark.py

```diff
@@ -366,8 +366,8 @@ def maybe_drop_added_dates(self, predicted, df):
         for df_name, df_i in df.groupby("ID"):
             predicted_i = predicted[predicted["ID"] == df_name].copy(deep=True)
             df_i["ds"] = convert_to_datetime(df_i["ds"])
-            df_i.set_index("ds")
-            predicted_i.set_index("time")
+            df_i.set_index("ds", inplace=True)
+            predicted_i.set_index("time", inplace=True)
```

Comment:
Small bugfix, as `pd.set_index` uses `inplace=False` as default, the pytests failed locally. Changed to `inplace=True` resolves the issue.